apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/PolicyDetailsV2.tsx

```diff
@@ -117,11 +124,10 @@ export const PolicyDetailsV2 = ({
             name="table"
             render={({ field }) => (
               <FormItem_Shadcn_ className="col-span-6 flex flex-col gap-y-1">
-                <FormLabel_Shadcn_ className="flex items-center gap-x-4">
+                <FormLabel_Shadcn_ className="flex items-center gap-x-1">
```

Comment:
these should still be gap-x-4
it should be like "Label", and then the descriptor, so not in the same sentence
e.g "Table", and "on clause"

Reply:
^ ahhh yes `on` and `clause` should be closer you're right

Reply:
with ref to prod atm
![image](https://github.com/supabase/supabase/assets/19742402/21736b0a-4571-429d-bf4d-efa7f1f385fd)

Reply:
Aaaah, I'll revert now, you're right

Reply:
Fixed.